src/create-render-context.js

```diff
@@ -157,7 +162,7 @@ const createRenderContext = function(
             // have it pretend that it is (it still isn't).
             pretendToBeVisual: true,
         },
-    );
+    ): any);
```

Comment:
We are adding some things to the JSDOM object. We don't want them to be optional because that would mean null checking just to keep flow happy, so instead, we cast here, knowing that for a few lines of code, the things will be missing.